zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -259,6 +268,24 @@ def get_user_avatar_source(image_url: str) -> str:
         avatar_source = 'U'
     return avatar_source
 
+def construct_avatar_url(slack_user_id: str, team_id: str, avatar_hash: str) -> str:
+    avatar_url = "https://ca.slack-edge.com/{}-{}-{}-512".format(team_id, slack_user_id,
```

Comment:
Why the 512 version? Would it reduce the bandwidth with a smaller version? Which size does Zulip use?

Reply:
@rht I am just testing this currently. I am able to get the normal avatars displayed, but unable to display avatar of the medium size with `-medium.png`  extension.